pkg/observability/metrics/metrics.go

```diff
@@ -28,6 +28,8 @@ type Recorder interface {
 	IncAdmissionReviewError(webhook, namespace, resource string, operation Operation, kind ReviewKind)
 	// ObserveAdmissionReviewDuration will observe the duration of a admission review.
 	ObserveAdmissionReviewDuration(webhook, namespace, resource string, operation Operation, kind ReviewKind, start time.Time)
+	// IncValidationReviewAllowed will increment in one the admission review allowed counter.
+	IncValidationReviewAllowed(webhook, namespace, resource string, operation Operation, allowed bool)
```

Comment:
I see a bit strange having `allowed` as a suffix in things that we also have `not allowed`. Does it make sense to you having `IncValidationReviewResult` instead `IncValidationReviewAllowed` and `validation_review_results_total` instead of `validation_review_allowed_total`?

Reply:
Check this out https://github.com/slok/kubewebhook/pull/65/commits/8995728d59f825595b4a02e747bc9b5d070caf31